### ISSUE TITLE
[FEATURE]: No UI scaling option

### ISSUE THREAD

**Issue by flabbet** [PixiEditor/PixiEditor#1107]
### Feature Description

This is a accessibility feature that is missing, some users may find PixiEditor UI too small on bigger screens. Ui scaling option is a simple solution to that.

### Steps how to use this feature

```bash
1. Set it in the settings to 50%, 100%, 200% etc
```

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

**Comment by rempas** [comment]
Please allow custom options that a lot of apps don't allow. Some users may want to scale for 135% for example. Give them the ability.

**Comment by rempas** [comment]
Hello! Any progress on this one? This and the very little documentation on nodes is what make the app unaccusable for me.

**Comment by flabbet** [comment]
@rempas I have this task queued, right now I am focused on brush engine, so I'll be able to start working on this after that. I added up for grabs tag, so maybe someone will work on it before me

**Comment by rempas** [comment]
> [@rempas](https://github.com/rempas) I have this task queued, right now I am focused on brush engine, so I'll be able to start working on this after that. I added up for grabs tag, so maybe someone will work on it before me

I understand. Thank you for your work! This app is extremely promising! I hope you get more attention from people that know how to help and more sponsors.

**Comment by jjhaggar** [comment]
Strong support for this feature. The current UI/menu text size makes PixiEditor effectively unusable for me (accessibility blocker). 

PR #1262 looks promising :) Ideally, the UI scale could allow custom values (e.g., 135%) and apply consistently across menus and dialogs.

**Comment by rempas** [comment]
> Strong support for this feature. The current UI/menu text size makes PixiEditor effectively unusable for me (accessibility blocker).
> 
> PR [#1262](https://github.com/PixiEditor/PixiEditor/pull/1262) looks promising :) Ideally, the UI scale could allow custom values (e.g., 135%) and apply consistently across menus and dialogs.

Damn, it looks good! Too bad it seems to be abandoned.

**Comment by jjhaggar** [comment]
> Damn, it looks good! Too bad it seems to be abandoned.

Hopefully the main dev (flabbet) will make their magic soon :) 
Last month I asked about this in Discord and they said they'd probably finish the PR (I hope it's not too hard, cause I really want to use the software).

**Comment by rempas** [comment]
> > Damn, it looks good! Too bad it seems to be abandoned.
> 
> Hopefully the main dev (flabbet) will make their magic soon :) Last month I asked about this in Discord and they said they'd probably finish the PR (I hope it's not too hard, cause I really want to use the software).

Me too! UI scaling and more docs/tutorial videos would be so appreciated and make me actually able to use the software.

**Comment by flabbet** [comment]
Implemented in 2.1.0.14-dev http://pixieditor.net/docs/open-beta/

**Comment by rempas** [comment]
> Implemented in 2.1.0.14-dev http://pixieditor.net/docs/open-beta/

Let's goooooo!!!! Finally, thank you so so much!!!

**Comment by rempas** [comment]
@flabbet I just tested and, the file dialog and the settings window do not scale.

**Comment by flabbet** [comment]
@rempas 

<img width="339" height="44" alt="Image" src="https://github.com/user-attachments/assets/54d36d76-69b6-4e05-8966-4926286cb592" />

What do you mean file dialog? The windows one?

**Comment by rempas** [comment]
> [@rempas](https://github.com/rempas)
> <img alt="Image" width="339" height="44" src="https://private-user-images.githubusercontent.com/25402427/564878099-54d36d76-69b6-4e05-8966-4926286cb592.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NzM3NjE4NzAsIm5iZiI6MTc3Mzc2MTU3MCwicGF0aCI6Ii8yNTQwMjQyNy81NjQ4NzgwOTktNTRkMzZkNzYtNjliNi00ZTA1LTg5NjYtNDkyNjI4NmNiNTkyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNjAzMTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjYwMzE3VDE1MzI1MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQyNTRmMjdjYmUzYjNlNjYyMTVjYjg3NTU5OTc3Mjg0ZjM1OGM4OThlYjlmZDhiZWNhNjEzZGZiMWY2NGIwY2QmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.KGXJeDv5mgsiQVTMHoFz_VlFrSMXMyPQciXC2Z7HlhA">
> 
> What do you mean file dialog? The windows one?

I'm on Linux. Here is how it looks:

<img width="2502" height="1324" alt="Image" src="https://github.com/user-attachments/assets/9eed71b9-eb94-4878-8f91-154b5890e338" />

**Comment by flabbet** [comment]
Ah, we have no control over that. That's a native linux file picker. I am afraid we can't do anything about it

**Comment by rempas** [comment]
> Ah, we have no control over that. That's a native linux file picker. I am afraid we can't do anything about it

Oh, I see, thank you. Might get fixed once AvaloniaUI ports to Wayland.
